checkstyle/suppressions.xml

```diff
@@ -321,15 +321,15 @@
     <suppress checks="CyclomaticComplexity"
               files="(ConsumerGroupMember|GroupMetadataManager|GeneralUniformAssignmentBuilder|GroupCoordinatorRecordSerde).java"/>
     <suppress checks="(NPathComplexity|MethodLength)"
-              files="(GroupMetadataManager|ConsumerGroupTest|ShareGroupTest|GroupMetadataManagerTest|GroupMetadataManagerTestContext|GeneralUniformAssignmentBuilder).java"/>
+              files="(GroupMetadataManager|ConsumerGroupTest|ShareGroupTest|GroupMetadataManagerTest|GroupMetadataManagerTestContext|GeneralUniformAssignmentBuilder|StreamsGroupMemberTest).java"/>
```

Comment:
are there rules that we follow for adding these exceptions?

Reply:
Not really.

I am normally reluctant to suppress warnings and try to fix the code, but in this case it seemed better to keep the code in one method.